src/ripple/app/paths/impl/BookStep.cpp

```diff
@@ -297,6 +297,13 @@ class BookPaymentStep : public BookStep<TIn, TOut, BookPaymentStep<TIn, TOut>>
         return true;
     }
 
+    // A payment doesn't use max quality threshold (limitQuality)
```

Comment:
You are right, AMM supports cross-currency Payment. The comment is not correct. I meant to say that the strand's limitQuality doesn't relate to the step's quality in case of Payment.